definition_validator/validator.cc

```diff
@@ -62,6 +62,8 @@ void validateCompatibleOverride(const core::GlobalState &gs, core::SymbolRef sup
                 e.setHeader("Implementation of abstract method `{}` must accept at least `{}` positional arguments",
                             superMethod.data(gs)->show(gs), leftPos);
                 e.addErrorLine(superMethod.data(gs)->loc(), "Base method defined here");
+                e.replaceWith("Accept the right arguments", method.data(gs)->loc(), "{}",
+                              superMethod.data(gs)->loc().source(gs));
```

Comment:
This could delete arguments that you've written, or rename them. I think we want to think carefully about these.